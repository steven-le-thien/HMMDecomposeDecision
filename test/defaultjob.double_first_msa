>L
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCAGGGAGATAGGCGAAGAAGCAAACACAACGCATCTCCGGCCGGAACCACTGCGTTTAAGCAAAGACCTCGGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATTGGTGGATGCGGTACTAGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGATGATGGAAAGGTACTCCCTATGGGGACTCGCCGCACGCTAGCGAGCCCGGCATTCGCAGTTAGGTTCCTTACAGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATCTTATGCGTGCGGGTATAGTTGCGTTAGTATATATTGCCTAAATGCTGTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACTATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGAACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCAGGCTTACGGCGATCCCAGTGGCCCCTAACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACTCAAGGATAAGTCCGGGTGAAGATTTCACGGTCTTACCCGCTTAGCCGGGGCGTTTCAACTAATTCGTTACTTCGTTCACGATGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATCGAATATCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAACGAATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
>K
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCATGGAGAAAGGCGAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAAGCAAAGACCTCGGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATTGGTGTATGCGGTACTAGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACTCCCTATGGGGACTCGCCGCACGCTAGCGAGCCGGGCATTCGCAGTTAGGTTGCTAACTGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGTTGCGTTAGTATATATTGCCTAAATGCTGTACAAATGTCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCGTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGCACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCAAGCTTACGGCGATCCCAGTGGCCCCTAACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACCCAAGGATAAGTCCGGGTGAAGATTTCACGGTCTTACCCGCTTAGCCGGGGCGTTTCAACTAATTCGTTACTTCCTTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATTGAATATCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAATGAATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
>H
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCATGGAGATAGGCGAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAAGCAAAGACCTCGGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGATTGTTGATGATTGGTGAATGCGGTACTAGGATCCAATACCTTAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACGCCCTATGGGGACTCGCCGCACGCTAGCGAGCCCGGCATTCGCAGTTAGGTTGCTAACAGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTTTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGTTGCGTTAGTATATATTGCCTAAATGCTGTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGGACAGCAGTTAGGTCTACTTGCAGCTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCAGGCTTACGGCGATCCCAGTGGCCCCCAACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACTCAAGGATAAGTCCGGGTGAAGATTTCACGGTCTTACCCGCTTAGCCGGGGCGTTTCAACTAATTCGTTACTTCCGTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATTGAATAGCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAATGGATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
