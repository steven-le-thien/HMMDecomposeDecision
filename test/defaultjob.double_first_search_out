#
# Program:         hmmsearch
# Version:         3.1b2 (February 2015)
# Pipeline mode:   SEARCH
# Query file:      defaultjob.double_first_hmm
# Target file:     defaultjob.double_first_msa
# Option settings: hmmsearch --tblout defaultjob.double_first_search_out --noali -E Infinity --max defaultjob.double_first_hmm defaultjob.double_first_msa 
# Current dir:     /Users/lethien_96/HMMDecomposeDecision/test
# Date:            Sun Jul 22 16:53:08 2018
# [ok]
