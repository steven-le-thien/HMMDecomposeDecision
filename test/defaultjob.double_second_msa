>J
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGGAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCATGGAGATAGGCGAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAGGCAAAGACCTCGGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATTGGTGTATACGGTACTAGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACTCCCTATGGGGACTCGCCGCACGCTAGCGAGCCCGGCATTCGCAGTTAGGTTGCTAACAGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGATGCGTTAGTATATATTGCCTAAATGCTGTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGAACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCTCCCTTACGGCGATCCCAGTGGCGCCTAACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACTCAAGGATAAGTCCGGGTGAAGATTTCACGGTCTTACCCGCTTAGCCGGGGCGTTTCAACTAATTCGTTACTTCCTTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAACAATTGAATATCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAATGAATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
>I
AGGACTTTCCACTCATTTCGGTGCCCAGGGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCATGGACATAGGCGAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAAGCAAAGACCTCCGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATTGGTGTATGCGGTACTAGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACTCCCTATGGGGACTCGCCGCACGCTAGCGAGCCCGGCATTCGCAGTTAGGTTGCTAACAGATTGTTGACAGCCCCTTACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTATGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGTTGCGTTAGTATATATTGCCTAAATGCTGTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGAACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCGGGCTTACGGCGATCCCAGTGGCCCCTAACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACTCAAGGATAAGTCCGGGTGAAGATTTCACGGTTTTACCCGCTTAGCCGGGGCGTTTCAAATAATTCGTTACTTCCTTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATTGAATATCGATCGTAGGTGAATGTGCAGACAGAGTTGAAAGTTAACGAATGAATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
>G
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTATTGTGCAGTTATACATTTCATGGAGATAGGCGAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAAGCAAAGACCTCGGCCCCATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATAGGTGTATGCGGTACTTGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACTCCCTATGGGGACTCGCCGCACGCTCGCGAGCCCGGCATTCGCAGTTAGGTTGCTAACAGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGTTGCGTTAGTATATATTCCCTAAATGCTGTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATGCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGAACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCAGGCTTACGGCGATCCCAGTGGCCCCTAACAGTAAGGCCTGGAATGAGTACTGGTGAACAAACTCAAGGATAAGTCCGGGTGAAGATTTCACGGTCTTACCCGCTTAGCCGGGGCTTTTCAACTAATTCGTTACTTCCTTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATTGAATATCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAATGAATTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGGCCT
>F
AGGACTTTCCACTCATTTCGGTGCCCAGCGCACGTTATGAGCGCGGTGTCCGTAGGACGAATCCGCAGGAAAGCTAGTGTGCAGTTATACATTTCATGGAGATAGGCCAAGAAGCAAACACAACGCATCTCAGGCCGGAACCACTGCGTTTAAGCAAAGACCTCGGCCCGATAACATGATCCTGAGCGAGAGTAGCGCCAGAAACCCTTCGGTCGGGTGAATGAATGTTGATGATTGGTGTATGCGGTACTAGGATCCAATACCTGAGCGGCGACAGCACCACCTAAGACGATGGAAAGGTACTCCTTATGGGGACTCGCCGCACGCTAGCGAGCCCGGCATTCGGAGTTAGGTAGCTAACAGATTGTTGACAGCCCCATACGCTCGTGTCGGACATGTCAGCGGCAGTATACTCTTTACGTTCCACAATGATCCGACCACCCGGTCGACAAATGTTATGCGTGCGGGTATAGTTGCGTTAATATATATTGCCTAAATGCTCTACAACTGGCTAGTAACGAGCCCGGGAAATGCGCGCTGATCTAAATAGGCACGTGGATACGGCTTGTCTTAGGAATCCTGTTCAGCCTGCGATGGTGGATATTAGTAACTGCGAACGACCATTTCCAGTAAAGTTCCTGGAACAGCCCTCAATTGAACAGCAGTTAGGTCTACTTGCAACTGGGCTATGCCCCTGCAGCTTACAACGAGAATTGACAGTTCAGGCTTACGGCGATCCCAGTGGCCCCTCACAGTAAGGCCTGGAATGAGTACTGGTCAACAAACTCAAGGATAAGTCCGGGTGATGATTTCACGGTCTTACCCGCTTAGCCGGGGCGTTTCAACTAATTCGTTACTTCCTTCACGTTGTGTCCGGCAGTTGGTAAGTTGCCAACCCGCAAGAATTGAATATCGATCGTAGGTGAATGTGCAGACAGAGTCGAAAGTTAACGAATGACTTTGCACCAGGGTTTACTCCGCCTAACGAATGCATCGTCCT
