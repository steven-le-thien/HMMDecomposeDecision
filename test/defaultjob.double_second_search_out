#
# Program:         hmmsearch
# Version:         3.1b2 (February 2015)
# Pipeline mode:   SEARCH
# Query file:      defaultjob.double_second_hmm
# Target file:     defaultjob.double_second_msa
# Option settings: hmmsearch --tblout defaultjob.double_second_search_out --noali -E Infinity --max defaultjob.double_second_hmm defaultjob.double_second_msa 
# Current dir:     /Users/lethien_96/HMMDecomposeDecision/test
# Date:            Sun Jul 22 16:53:09 2018
# [ok]
