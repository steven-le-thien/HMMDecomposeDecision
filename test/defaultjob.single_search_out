#
# Program:         hmmsearch
# Version:         3.1b2 (February 2015)
# Pipeline mode:   SEARCH
# Query file:      defaultjob.single_hmm
# Target file:     o
# Option settings: hmmsearch --tblout defaultjob.single_search_out --noali -E Infinity --max defaultjob.single_hmm o 
# Current dir:     /Users/lethien_96/HMMDecomposeDecision/test
# Date:            Sun Jul 22 16:53:08 2018
# [ok]
