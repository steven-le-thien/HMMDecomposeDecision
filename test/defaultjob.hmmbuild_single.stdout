# hmmbuild :: profile HMM construction from multiple sequence alignments
# HMMER 3.1b2 (February 2015); http://hmmer.org/
# Copyright (C) 2015 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# input alignment file:             o
# output HMM file:                  defaultjob.single_hmm
# input alignment is asserted as:   DNA
# sym fraction for model structure: 0.000
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

# idx name                  nseq  alen  mlen     W eff_nseq re/pos description
#---- -------------------- ----- ----- ----- ----- -------- ------ -----------
1     o                        7  1000  1000  1259     1.30  0.451 

# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.13
